### PR TITLE
Fix documentation

### PR DESCRIPTION
**Pull request by Neradoc**
Missing new lines for parameters list.
Lists all submodules in APIs.

### CODE REVIEW

File: adafruit_wiznet5k/adafruit_wiznet5k_ntp.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 # SPDX-License-Identifier: MIT
 
 """
-`wiznet5k_ntp`
```
Comment (tekktrik) [inline adafruit_wiznet5k/adafruit_wiznet5k_ntp.py:6]: Nice catch!

File: adafruit_wiznet5k/adafruit_wiznet5k_socket.py [hunk 2]
```diff
@@ -355,10 +365,11 @@ def recv_into(self, buf, nbytes=0, flags=0):
 
     def recvfrom_into(self, buf, nbytes=0, flags=0):
         """Reads some bytes from the connected remote address info the provided buffer.
+
         :param bytearray buf: Data buffer
         :param nbytes: Maximum number of bytes to receive
         :param int flags: ignored, present for compatibility.
-        :returns a tuple (nbytes, address) where address is a tuple (ip, port)
```
Comment (tekktrik) [inline adafruit_wiznet5k/adafruit_wiznet5k_socket.py:361]: Double nice catch!

Comment (tekktrik) [inline adafruit_wiznet5k/adafruit_wiznet5k_socket.py:361]: I believe this should be `:return:`

**Review by brentru — APPROVED** [review]

**Review by tekktrik — CHANGES_REQUESTED** [review]
Nice catches!  Only thing I noticed is that `:returns:` should be `:return:` I believe.  I think that goes for all the uses of `:returns:` in this library, unfortunately.

**Comment by Neradoc** [comment]
`:returns:` renders the same as `:return:` so I didn't touch them, but I agree, they should be consistent. I'll do that.

**Comment by tekktrik** [comment]
> `:returns:` renders the same as `:return:` so I didn't touch them, but I agree, they should be consistent. I'll do that.

Oh, good to know!

**Comment by tekktrik** [comment]
@Neradoc looks like the new black wants to reformat something, wanna run `pre-commit` again?

**Review by tekktrik — APPROVED** [review]

**Pull request merged**